packages/tasks_panel/src/utils/swa.ts

```diff
@@ -1,6 +1,9 @@
-import { SWATracker } from "@sap/swa-for-sapbas-vsx";
+// import { SWATracker } from "@sap/swa-for-sapbas-vsx";
 
-export type ISWATracker = Pick<SWATracker, "track">;
+// export type ISWATracker = Pick<SWATracker, "track">;
```

Comment:
why you had to change that?

Reply:
my understanding is 'swa' library is deprecated